vault/init.sls

```diff
@@ -1,8 +1,20 @@
-{%- set version = salt.pillar.get('vault:vault_version', '0.7.0') %}
-{%- set source_hash = salt.pillar.get('vault:source_hash', 'c6d97220e75335f75bd6f603bb23f1f16fe8e2a9d850ba59599b1a0e4d067aaa') %}
-install Vault:
-  archive.extracted:
-    - name: /usr/local/bin
-    - source: https://releases.hashicorp.com/vault/{{ version }}/vault_{{ version }}_linux_amd64.zip
-    - source_hash: {{ source_hash }}
-    - enforce_toplevel: False
+{% from "vault/map.jinja" import vault with context %}
+# using archive.extracted causes: 'Comment: Failed to cache https://releases.hashicorp.com/vault/0.7.0/vault_0.7.0_linux_amd64.zip: [Errno 1] _ssl.c:493: error:1409442E:SSL routines:SSL3_READ_BYTES:tlsv1 alert protocol version'
```

Comment:
anythin gthat uses python's http.py like archive.extracted, if we use 3rd party compiled tools like curl it's avoided